lib/tasks/demo_seed.rake

```diff
@@ -326,7 +326,7 @@ namespace :demo  do
           :expires_at         => Time.zone.now+1.year,
           :name_on_card       => 'Paul PI',
           :expiration_month   => '10',
-          :expiration_year    => '2014',
+          :expiration_year    => '2020',
```

Comment:
Sure thing. Also, I was planning on using `5.years.from_now.year` so my seeded credit card doesn't expire for a while.